R/bundle.R

```diff
@@ -625,6 +627,14 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters,
       info <- as.list(deps[i, c('Source',
                                 'Repository')])
 
+      if (is.na(info$Repository)) {
+        if (tolower(info$Source) %in% c("github", "bitbucket", "source")) {
```

Comment:
I don't love this copy-paste of the SCM types; I'll extract this into a helper function.